pkgs/native_toolchain_c/lib/src/cbuilder/run_cbuilder.dart

```diff
@@ -197,8 +217,12 @@ class RunCBuilder {
     final result = await runProcess(
       executable: compiler.uri,
       arguments: [
+        if (std != null) '/std:$std',
+        if (language == Language.cpp) '/TP',
```

Comment:
Are we missing a use of `cppLinkStdLib` here?

Reply:
As far as I can tell, there is no comparable option for MSVC to link with a specific standard library.

Reply:
Okay.

In my testing you always have to provide a path to the OS APIs and stdlib via the `vcvars` script above. Since we inherit these from visual studio, maybe there's no swapping out.

Maybe add an `ArgumentError` for supplying a `cppLinkStdLib` if target os is Windows.

Reply:
Done